junit-platform-engine/src/main/java/org/junit/platform/engine/support/hierarchical/ParallelExecutionConfiguration.java

```diff
@@ -60,4 +61,16 @@
 	 */
 	int getKeepAliveSeconds();
 
+	/**
+	 * Get the predicate called when there the ForkJoinPool is saturated.
```

Comment:
```suggestion
	 * Get the predicate called when the {@code ForkJoinPool} is saturated.
```
(Same remark for classes below)